### PR TITLE
dot input calendar test 

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 1]
```diff
@@ -1,119 +1,255 @@
...
+
+            it('should render and not break when is a unexpected value and set default(text)', async () => {
+                element.setProperty('type', { test: true });
+                await page.waitForChanges();
+                expect(input.getAttribute('type')).toBe('[object Object]');
             });
         });
+    });
+
+    describe('@Events', () => {
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: missing `requiredMessage` test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 2]
```diff
@@ -1,119 +1,255 @@
...
+                expect(input.getAttribute('disabled')).toBeDefined();
+            });
+
+            it('should not render disabled attribute', async () => {
+                element.setProperty('disabled', 'false');
+                await page.waitForChanges();
+                expect(input.getAttribute('disabled')).toBeNull();
+            });
+
+            it('should render disabled attribute with invalid value', async () => {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: remove this test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 3]
```diff
@@ -1,119 +1,255 @@
...
-        it('should mark as touched when onblur', async () => {
-            await input.triggerEvent('blur');
-            await page.waitForChanges();
+        describe('min', () => {
+            it('should set value correctly', async () => {
+                element.setProperty('min', '111');
+                await page.waitForChanges();
+                expect(await input.getAttribute('min')).toBe('111');
+            });
+            it('should render and not break when is a unexpected value', async () => {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: remove this test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 4]
```diff
@@ -1,119 +1,255 @@
...
-                    dotTouched: true,
-                    dotValid: true
-                }
+        describe('max', () => {
+            it('should set value correctly', async () => {
+                element.setProperty('max', '9');
+                await page.waitForChanges();
+                expect(await input.getAttribute('max')).toBe('9');
+            });
+            it('should render and not break when is a unexpected value', async () => {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: remove this test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 5]
```diff
@@ -1,119 +1,255 @@
...
+            it('should set value default value correctly', async () => {
+                await page.waitForChanges();
+                expect(await input.getAttribute('step')).toBe('1');
+            });
+            it('should set value correctly', async () => {
+                element.setProperty('step', '2');
+                await page.waitForChanges();
+                expect(await input.getAttribute('step')).toBe('2');
+            });
+            it('should render and not break when is a unexpected value', async () => {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: remove this test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 6]
```diff
@@ -1,119 +1,255 @@
...
-                message: 'Time out of range',
-                show: false
+        describe('type', () => {
+            it('should set value correctly', async () => {
+                element.setProperty('type', 'time');
+                await page.waitForChanges();
+                expect(input.getAttribute('type')).toBe('time');
+            });
+
+            it('should render and not break when is a unexpected value and set default(text)', async () => {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: remove this test

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 7]
```diff
@@ -74,38 +69,32 @@ describe('dot-input-calendar', () => {
                 await page.waitForChanges();
                 expect(input.getAttribute('disabled')).toBeNull();
             });
+        });
 
-            it('should render disabled attribute with invalid value', async () => {
-                element.setProperty('disabled', { test: 'test' });
+        describe('min', () => {
+            it('should not render  attribute by default', async () => {
                 await page.waitForChanges();
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: if you're not changing anything I think this is not needed

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts [hunk 8]
```diff
@@ -74,38 +69,32 @@ describe('dot-input-calendar', () => {
...
```
Comment (alfredo-dotcms) [inline projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: https://github.com/dotCMS/core-web/pull/1042/commits/5c4e1b2aae769867493cbcabcf728eb044685bc6#r289175026

File: projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts (hunk 7 above)
Comment (hmoreras) [reply projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts]: is just to make sure that the default value is empty.

**Pull request merged**